pkg/config/setup/config.go

```diff
@@ -684,6 +684,7 @@ func InitConfig(config pkgconfigmodel.Config) {
 	config.BindEnvAndSetDefault("admission_controller.add_aks_selectors", false) // adds in the webhook some selectors that are required in AKS
 	config.BindEnvAndSetDefault("admission_controller.auto_instrumentation.enabled", true)
 	config.BindEnvAndSetDefault("admission_controller.auto_instrumentation.endpoint", "/injectlib")
+	config.BindEnvAndSetDefault("admission_controller.auto_instrumentation.dotnet_profiler.enabled", false)
```

Comment:
I would very much prefer if we didn't add another setting specifically for .NET. We'll have to support this setting which adds overhead to making changes going forward.

Instead, if we really wanted to backwards support .NET profiling users, we can check if `DD_PROFILING_ENABLED` is set and then inject the .NET LD_PRELOAD if so. Currently users have to set that variable to enable profiling for .NET today.